commit f0f416bbdefaeb936b4848864f18f86830cc0989
Author: Chanwoo Choi <cw00.choi@samsung.com>
Date:   Tue Jan 31 15:38:17 2017 +0900

    PM / devfreq: Fix wrong trans_stat of passive devfreq device
    
    commit 30582c25a4b4e0a5e456a309fde79b845e9473b2 upstream.
    
    Until now, the trans_stat information of passive devfreq is not updated.
    This patch updates the trans_stat information after setting the target
    frequency of passive devfreq device.
    
    Fixes: 996133119f57 ("PM / devfreq: Add new passive governor")
    Signed-off-by: Chanwoo Choi <cw00.choi@samsung.com>
    Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1261_f0f416bbdefaeb936b4848864f18f86830cc0989.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1261_f0f416bbdefaeb936b4848864f18f86830cc0989.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
